### ISSUE TITLE
Social media icons should have a label title for the screenreader

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#594]
- [x] Social media icons should have a label "Social Media" only for the screenreader, not visible.

https://swiss.github.io/styleguide/de/footer.html#04-footer-02-social-media

![image](https://user-images.githubusercontent.com/7510842/32946771-1b6cf3c2-cb99-11e7-9eda-099b3bfa3389.png)

- [x] Please add a label also for "Social Media  Sharing"

https://swiss.github.io/styleguide/de/functions.html#04-functions-02-social-sharing

**Comment by gillerr** [comment]
Simply add a label with `sr-only` class to the `nav` element

**Comment by bit-pro-iew-eui** [comment]
Please don't use H1-6 use another html-tag which is read by at least NVDA
